app/scripts/lib/router.js

```diff
@@ -98,12 +101,14 @@ define(function (require, exports, module) {
       'settings/display_name(/)': createChildViewHandler(DisplayNameView, SettingsView),
       'signin(/)': createViewHandler(SignInView),
       'signin_complete(/)': createViewHandler(ReadyView, { type: 'sign_in' }),
+      'signin_confirmed(/)': createViewHandler(ReadyView, { type: 'sign_in_confirmed' }),
       'signin_permissions(/)': createViewHandler(PermissionsView, { type: 'sign_in' }),
       'signup(/)': createViewHandler(SignUpView),
       'signup_complete(/)': createViewHandler(ReadyView, { type: 'sign_up' }),
       'signup_permissions(/)': createViewHandler(PermissionsView, { type: 'sign_up' }),
       'unexpected_error(/)': createViewHandler(UnexpectedErrorView),
-      'verify_email(/)': createViewHandler(CompleteSignUpView)
+      'verify_email(/)': createViewHandler(CompleteSignUpView),
+      'verify_sign_in(/)': createViewHandler(CompleteSignInView)
```

Comment:
I'm wondering if this route should be called `verify_token` because it will eventually be used for more than just sign-in. It'll probably also be used for change password and delete account.